test/grouping.jl

```diff
@@ -1448,10 +1448,11 @@ end
           groupby_checked(DataFrame(x2_identity=[1,1,2]), [])
     @test isequal_typed(DataFrame(gdf), df)
 
-    @test sprint(show, groupby_checked(df, [])) == """
-        GroupedDataFrame with 1 group based on key: 
-        Group 1 (3 rows): 
-         Row │ x1     x2     y
+    # work around automatic trimming of trailing whitespace in most editors
```

Comment:
Is this the only place where we have this problem?

Reply:
we do not need to hunt for it. My editor will catch it the next time I edit some file that has this problem and I will fix it.